bin/dbsync

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+var path   = require('path');
+var fs     = require('fs');
+var coffee = require('coffee-script');
+var src    = path.join(path.dirname(fs.realpathSync(__filename)), '../src');
+
```

Comment:
The reason I am suggestion running the lint in the binary is because this file is not really built. It is just run. So when would someone automagically lint this if they just run it? I guess some gulp would need to be put in to just `lint, watch, run` .